debug-ui/lib/address-labels.ts

```diff
@@ -0,0 +1,109 @@
+import { Connection, PublicKey } from '@solana/web3.js';
+import { Metaplex } from '@metaplex-foundation/js';
+import { TOKEN_2022_PROGRAM_ID } from '@solana/spl-token';
+import { ENV, TokenInfo, TokenListProvider } from '@solana/spl-token-registry';
+import { Market } from '@cks-systems/manifest-sdk';
+import { LabelsByAddr } from './types';
+import { Dispatch, SetStateAction } from 'react';
+
+export const getTokenSymbol = async (
```

Comment:
TODO to add a cache. With more markets, we will get rate limited

Reply:
this only runs once on startup for the ui. This already uses a cache. when the app loads in the user's browser it only gets fetched once. or did you mean some sort of local storage cache? or some server that caches it?